neo/Network/NeoNode.py

```diff
@@ -128,7 +130,12 @@ def connectionLost(self, reason=None):
             if self.block_loop_deferred:
                 self.block_loop_deferred.cancel()
             if self.peer_loop_deferred:
-                self.block_loop_deferred.cancel()
+                self.peer_loop_deferred.cancel()
+
+            if self.block_loop:
```

Comment:
I believe we have to check the `running` flag before calling Stop or we get an assert error
```suggestion
            if self.block_loop.running:
```